crates/raw-vaas/src/payloads/token_bridge/gov.rs

```diff
@@ -27,16 +29,20 @@ impl<'a> TokenBridgeGovPayload<'a> {
         self.span
     }
 
-    pub fn decree(&self) -> TokenBridgeDecree<'a> {
-        self.decree
+    pub fn decree(&self) -> &TokenBridgeDecree<'a> {
+        &self.decree
```

Comment:
any reason for this change? this likely saves nothing, as TokenBridgeDecree is a fat pointer.

Reply:
Good call, I can change it back.

I made this change to mirror this in the token bridge payload submodule:
```
    pub fn message(&self) -> &TokenBridgeMessage<'a>
```

Should I make this just return TokenBridgeMessage to make things consistent?

Reply:
I just made this change here: 51162146815f1fee4bf2d3e12348ce08de92fc7d